sinbin/src/Pos/Sinbin/Reporting/Wlog.hs

```diff
@@ -1,136 +0,0 @@
-    charsConst :: Int
-    charsConst = 1024 * 1024 * 2
-    takeGlobalSize :: Int -> [Text] -> [Text]
-    takeGlobalSize _ [] = []
-    takeGlobalSize curLimit (t:xs) =
-        let delta = curLimit - length t
-        in bool [] (t : (takeGlobalSize delta xs)) (delta > 0)
-
--- | Given logger config, retrieves all (logger name, filepath) for
--- every logger that has file handle. Filepath inside does __not__
--- contain the common logger config prefix.
-retrieveLogFiles :: LoggerConfig -> [([LoggerName], FilePath)]
-retrieveLogFiles lconfig = fromLogTree $ lconfig ^. lcTree
-  where
-    fromLogTree lt =
-        let curElems = map ([],) (lt ^.. ltFiles . each . hwFilePath)
-            iterNext (part, node) = map (first (part :)) $ fromLogTree node
-        in curElems ++ concatMap iterNext (lt ^. ltSubloggers . to HM.toList)
-
--- | Pass a list of absolute paths to log files. This function will
--- archive and compress these files and put resulting file into log
--- directory (returning filepath is absolute).
---
--- It will throw a PackingError in case:
---   - Any of the file paths given does not point to an existing file.
---   - Any of the file paths could not be converted to a tar path, for instance
---     because it is too long.
-compressLogs :: [FilePath] -> IO FilePath
-compressLogs files = do
-    tar <- tarPackIndependently files
-    tarxz <-
-        BS.concat <$>
-        runConduitRes (yield tar .| Lzma.compress (Just 0) .| consume)
-    aName <- getArchiveName
-    withFile aName WriteMode $ \handle -> do
-        BS.hPut handle tarxz
-        hFlush handle
-    pure aName
-  where
-    tarPackIndependently :: [FilePath] -> IO ByteString
-    tarPackIndependently paths = do
-        entries <- forM paths $ \p -> do
-            unlessM (doesFileExist p) $ throwM $
-                PackingError $ "can't pack log file " <> fromString p <>
-                               " because it doesn't exist or it's not a file"
-            tPath <- either (throwM . PackingError . fromString)
-                            pure
-                            (Tar.toTarPath False $ takeFileName p)
-            pabs <- canonicalizePath p
-            Tar.packFileEntry pabs tPath
-        pure $ BSL.toStrict $ Tar.write entries
-    getArchiveName = do
-        curTime <- formatTime defaultTimeLocale "%q" <$> getCurrentTime
-        tempDir <- getTemporaryDirectory
-        pure $ tempDir <//> ("report-" <> curTime <> ".tar.lzma")
-
--- | Creates a temp file from given text
-withTempLogFile :: Text -> (FilePath -> IO a) -> IO a
-withTempLogFile rawLogs action = do
-    withSystemTempFile "main.log" $ \tempFp tempHandle -> do
-        let getArchivePath = do
-                TIO.hPutStrLn tempHandle rawLogs
-                hClose tempHandle
-                archivePath <- compressLogs [tempFp]
-                canonicalizePath archivePath
-            removeArchive = removeFile
-        bracket getArchivePath removeArchive action
```

Comment:
Module moved to [here](https://github.com/input-output-hk/cardano-sl/pull/3254/files#diff-f2bc6a0e5b7400a041e228f7abd42b0bR136) and renamed.